test/unit/bps_tree.cc

```diff
@@ -61,9 +59,9 @@ compare(type_t a, type_t b);
 #undef bps_tree_arg_t
 
 /* true tree with true settings */
+#define test_TREE_EXTENT_SIZE 2048 /* value is to low specially for tests */
```

Comment:
I'd use `TREE_EXTENT_SIZE_test` similar to EE version here in the other places.

Reply:
There're aleady `bx1_TREE_EXTENT_SIZE` and `bx4_TREE_EXTENT_SIZE` in the CE BPS vector test, so I've made the name consistent with these.